vita3k/lang/include/lang/state.h

```diff
@@ -266,7 +266,7 @@ struct LangState {
         { "red", "Red" },
         { "green", "Green" },
         { "blue", "Blue" },
-        { "not_connected", "No compatible controllers connected.\nConnect a controller that is compatible with SDL2." },
+        { "not_connected", "No compatible controllers connected.\nConnect a controller that is compatible with SDL3." },
```

Comment:
Can you update all the translations files as well? For example: https://github.com/Vita3K/Vita3K/blob/1e15a4b324ad6a44a55253f6c65acdf8c9da23c0/lang/system/en-gb.xml#L297

Reply:
Done